about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="@codedit334">
+    <meta name="description" content="Capstone project 1: Camping website">
+    <meta name="keywords" content="Camp, Website, Responsive">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style/style.css">
+    <title>About All In Campsters</title>
+</head>
+
+<body>
+    <main>
+        <div class="navBar">
+            <div class="logo">
+                <a href="./index.html"><img src="./assets/AIC.svg" alt="Logo"></a>
+                
+            </div>
+            <div class="space"></div>
+            <ul class="menu">
+                <li class="x">&times;</li><br>
+                <li ><a href="./index.html">Index</a></li>
+                <li class="active"><a href="./about.html">About</a></li>
+            </ul>
+        </div>
+        <header class="about-header">
+            <nav>
+                <img src="./assets/ham.png" class="ham" alt="menu">
+            </nav>
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly you missed to implement `First` navbar and `main` navbar's other sections, please look at the picture below and please implement it 🙏🏼 [E.X] In this milestone you should keep original layout [design provided by Microverse](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 😃

![image](https://user-images.githubusercontent.com/63915024/217052659-7f7099a5-691c-41c3-b838-fb0a3196bef5.png)